project/Rat.scala

```diff
@@ -40,7 +40,7 @@ object Rat {
     file("daffodil-cli/src/windows/dialog.bmp"),
 
     // generated_code.[ch] examples
-    file("daffodil-runtime2/src/test/c/examples"),
+    file("daffodil-codegen-c/src/test/c/examples"),
```

Comment:
The osgi-refactor script has a Rat.scala file that references "runtime2". How do we want to handle that? I assume no changes and it's only there for historical reasons?

Reply:
I know, I saw the runtime2 still mentioned in the scripts too.  I left the scripts alone because they're historical artifacts that @mbeckerle wanted to keep around in case someone needs to develop new refactoring scripts again.  It would've been easy enough to get them out of git history, but only for someone who remembers that the scripts were there.  I have no objections changing runtime2 in them just to have a clean slate, however.

Reply:
I don't think it's necessary to make the changes. If the goal is to be historical, we probably do want it to match what it was as the time of use. Just wanted to bring it up since it was a place I saw runtime2 stuff.